src/display/process/allProcs.go

```diff
@@ -278,5 +275,15 @@ func AllProcVisuals(dataChannel chan []*proc.Process,
 		case <-tick: // Update page with new values
 			ui.Render(myPage.Grid)
 		}
+
+		if killSelected {
+			exists, _ := proc.PidExists(pidToKill)
+			if !exists {
+				runAllProc = true
+				killSelected = false
+				myPage.BodyList.SelectedRowStyle = selectedStyle
+				updateProcs()
+			}
+		}
 	}
```

Comment:
@Samyak2 This method of checking is accurate but the issue is we'll never get to this section of code as the above statements are in an infinite loop. The only time we exit from that loop is when we exit the program entirely! What you're doing in the else clause of `if !killSelected` suffices I feel.

Reply:
This is inside the loop xD, right after the `select`

Reply:
Oops, my bad!